Basic Note Application/Scenes/Auth/Register/RegisterViewController.swift

```diff
@@ -54,31 +63,30 @@ class RegisterViewController: UIViewController {
         setupViews()
     }
     
-    private let viewModel: RegisterViewModel
-    
-    init(viewModel: RegisterViewModel) {
-        self.viewModel = viewModel
-        super.init(nibName: nil, bundle: nil)
-    }
-    
-    required init?(coder: NSCoder) {
-        fatalError("init(coder:) has not been implemented")
-    }
-    
     private func setupViews() {
-        view.addSubview(stackView)
-        stackView.edgesToSuperview( insets: .init(top: 100, left: 10, bottom: 30, right: 10) ,usingSafeArea: true)
+        view.addSubview(scrollView)
+        scrollView.edgesToSuperview(usingSafeArea: true)
         
-        stackView.addArrangedSubview(authSignUpView)
+        view.addSubview(signInNowLabel)
```

Comment:
eklenen componentler yukarıdan aşağıya doğru yazılmalı, ayrıca scroll bottom verilmemeli
signInNowLabel.topToBottom(of: scrollView) diyerek signInNowLabel top constrainti scrollView bottom'a bağlanmalı.

Bu şekilde scroll üzerine label eklenmiş